### ISSUE TITLE
How to find my smart phone with flutter_blue?

### ISSUE THREAD

**Issue by martin284** [pauldemarco/flutter_blue#701]
I'm very new to Bluetooth. I want to write a small app where I can test the functionalities of the package flutter_blue. I installed the official example here https://pub.dev/packages/flutter_blue/example on my tablet and tried to find my smart phone but it could not be found. It seems to be a stupid question but do I have to do something special so that my smart phone is recognized as a server by the tablet?

**Comment by emretufekci** [comment]
This library does not support Classical Bluetooth instead of they're supporting BLE.

So, you need one central and one peripheral device

phones are central and with BLE technology it is not possible to connect to central device each other. So you need to transform one device to peripheral.

Use this library: https://pub.dev/packages/flutter_ble_peripheral

**Comment by martin284** [comment]
Thank you for your helpful answer. I tried the example of the flutter_ble_peripheral package but still my phone can't be found. Are you sure that the package is necessary to just find (not connect) the phone?

**Comment by emretufekci** [comment]
Yes it is necessary because of you need one peripheral point at least.

So one device should be central and other one should be peripheral. By using that package you can transform. If even you cannot see please make sure GPS & Bluetooth are enabled.

**Comment by vipinkashyap** [comment]
You could instead install a bluetooth simulator app on a spare phone/device and use that for now as your beacon.

**Comment by martin284** [comment]
Yes, I tried the BLE Peripheral Simulator and it works for me. Thanks.

**Comment by jahnli** [comment]
> 是的，我尝试了 BLE 外设模拟器，它对我有用。谢谢。

I would like to know if you have implemented it and if you can contribute some code
